src/components/app.js

```diff
@@ -45,26 +45,22 @@ const Routes = () => (
 			.filter(plugin => plugin.additionalRoutes)
 			.map(plugin => plugin.additionalRoutes)
 			.flat()
-			.map(([path, Component]) => {
```

Comment:
Having lots of linting problems, `npx eslint --fix src` doesn't seem to be working correctly. Any tips @germanferrero?

Reply:
@luandro I've not paid to much attention to linting lately. Maybe the way to go is to properly configure https://prettier.io/ for the project, and do a massive linting PR. Do you have any experience with it?

Reply:
I've got some experience with setting up Prettier + Husky for pre-commit linting. I could create another branch/PR for that.

Reply:
That would be awesome!!!